src/components/widget/widget.tmpl.html

```diff
@@ -1,6 +1,6 @@
 <div class="top-line">
   <div common-editable-title parent-widget="widget" class="editable-title" on-toggle="toggleEditTitle()" />
-  <div common-top-buttons parent-widget="widget" on-refresh="showWidget" ng-hide="isInfoPanelDisplayed() || isTitleEdited() || pdfMode" on-toggle-info-panel="toggleInfoPanel()"/>
+  <div common-top-buttons parent-widget="widget" on-refresh="showWidget" ng-hide="isTitleEdited() || pdfMode" on-toggle-info-panel="toggleInfoPanel()" is-info-panel-displayed="isInfoPanelDisplayed()"/>
```

Comment:
@an-pavlovych please keep wording consistent: something like `on-display` rather than `is-info-panel-displayed`